test_conformance/c11_atomics/common.h

```diff
@@ -800,23 +794,31 @@ std::string CBasicTest<HostAtomicType, HostDataType>::KernelCode(cl_uint maxNumD
     "\n";
   if(LocalMemory())
   {
-    code +=
-      "  // initialize atomics not reachable from host (first thread is doing this, other threads are waiting on barrier)\n"
-      "  if(get_local_id(0) == 0)\n"
-      "    for(uint dstItemIdx = 0; dstItemIdx < numDestItems; dstItemIdx++)\n"
-      "    {\n";
-    if(aTypeName == "atomic_flag")
-    {
-      code +=
-        "      if(finalDest[dstItemIdx])\n"
-        "        atomic_flag_test_and_set(destMemory+dstItemIdx);\n"
-        "      else\n"
-        "        atomic_flag_clear(destMemory+dstItemIdx);\n";
-    }
+      // memory_order_relaxed is sufficient for these initialization operations
+      // as the barrier below will act as a fence, providing an order to the
+      // operations. memory_scope_work_group is sufficient as local memory is
+      // only visible within the work-group.
+      code += "  // initialize atomics not reachable from host (first thread "
+              "is doing this, other threads are waiting on barrier)\n"
+              "  if(get_local_id(0) == 0)\n"
+              "    for(uint dstItemIdx = 0; dstItemIdx < numDestItems; "
+              "dstItemIdx++)\n"
+              "    {\n";
+      if (aTypeName == "atomic_flag")
+      {
+          code += "      if(finalDest[dstItemIdx])\n"
+                  "        "
+                  "atomic_flag_test_and_set_explicit(destMemory+dstItemIdx, "
+                  "memory_order_relaxed, memory_scope_work_group);\n"
+                  "      else\n"
+                  "        atomic_flag_clear_explicit(destMemory+dstItemIdx, "
+                  "memory_order_relaxed, memory_scope_work_group);\n";
```

Comment:
I know the formatting was done by clang-format, but I wonder if these strings would be easier to read if they were raw string literals?  No action required, just pondering ways to make these tests easier to understand.

Reply:
Good point! Yes, definitely, I have pushed a patch that replace the strings touched in this PR with raw strings.